src/modules/orbitdb/indexes/sql.js

```diff
@@ -39,7 +39,7 @@ class ChluSQLIndex extends ChluAbstractIndex {
         this.chluIpfs.logger.debug('ChluDB SQL Index: Defining Models')
         this.ReviewRecord = this.sequelize.define('reviewrecord', {
             data: Sequelize.JSONB,
-            latestVersion: Sequelize.STRING,
+            latestVersionData: Sequelize.JSONB,
```

Comment:
@fazo96 just to be sure, we aren't keeping this in the inmemory index? I assume we aren't.

Reply:
No, InMemory index only deals with the minimum amount of data possible (review multihashes only, no content)